Library/include/CSP/Systems/Users/UserSystem.h

```diff
@@ -222,6 +225,15 @@ class CSP_API CSP_NO_DISPOSE UserSystem : public SystemBase
 												  const csp::common::Optional<csp::common::String>& InRedirectUrl,
 												  NullResultCallback Callback);
 
+	/// @brief Get the Customer Portal Url for a user from Stripe
+	/// @param UserId csp::common::String : the id of the user associated with the customer portal
+	/// @param Callback StringResultCallback : callback that contains the customer portal URL of the User
+	CSP_ASYNC_RESULT void GetCustomerPortalUrl(const csp::common::String& UserId, CustomerPortalUrlResultCallback Callback);
+
+	/// @brief Get the checkout session Url for a user from Stripe
+	/// @param Tier csp::systems::TierNames : the tier of the checkout session needed
+	/// @param Callback CheckoutSessionUrlResultCallback : callback that contains the customer portal URL of the User
```

Comment:
Looks like the description of `CheckoutSessionUrlResultCallback` is incorrect here.